README.md

```diff
@@ -70,3 +76,38 @@ cd apps/blockchain && mix test test/blockchain/state_test.exs
 # Ethereum Transaction tests
 cd apps/blockchain && mix test test/blockchain/transaction_test.exs
 ```
+
+# Documentation
+To view module and reference documentation:
+
+1. Generate documentation.  
+`mix docs`
+
+2. View the generated docs.  
+`open doc/index.html`
+
+
+# License
+
+[![License: LGPL v3.0](https://img.shields.io/badge/License-LGPL%20v3-blue.svg)](https://www.gnu.org/licenses/lgpl-3.0)
+
+This project is licensed under the GNU Lesser General Public License v3.0. See the [LICENSE](LICENSE) file for details.
+
+
+# Contributing
+
+See the [CONTRIBUTING](CONTRIBUTING.md) document for contribution, testing and pull request protocol.
+
+
+# References
+
+* [Ethereum yellow paper](https://ethereum.github.io/yellowpaper/paper.pdf)(ETHEREUM: A SECURE DECENTRALISED GENERALISED TRANSACTION LEDGER
+BYZANTIUM VERSION )
+
+* [Message Calls in Ethereum](http://www.badykov.com/ethereum/2018/06/17/message-calls-in-ethereum/)
+
+Additional Ethereum Implementations
+
+* [Parity](https://github.com/paritytech/parity-ethereum)
+* [Geth](https://github.com/ethereum/go-ethereum/)
+* [EthereumJS](https://github.com/ethereumjs/ethereumjs-vm)
```

Comment:
Not sure if we want to include `py-evm` in here? https://github.com/ethereum/py-evm

Reply:
Sure, I will add.